cmd/osbuild-dnf-json-tests/main_test.go

```diff
@@ -31,13 +36,13 @@ func TestFetchChecksum(t *testing.T) {
 
 	repoCfg := rpmmd.RepoConfig{
 		Name:      "repo",
-		BaseURL:   fmt.Sprintf("file://%s", dir),
+		BaseURL:   "http://localhost:9000",
```

Comment:
The thing is, because the `dnf-json` service has its own private `/tmp` it could not access the files written at `dir` location. So I chose to keep the private `/tmp `and go this way for the test. I think the likelihood of a failure for the 9000 port is low, but I can enhance the log message in case the http server doesn't want to start to help a future debug.

Reply:
Thanks for explanation. this is fine!